bigquery_etl/metadata/validate_metadata.py

```diff
@@ -0,0 +1,66 @@
+"""Validate metadata files."""
+
+from argparse import ArgumentParser
+import logging
+import os
+import sys
+
+from .parse_metadata import Metadata
+from ..util import standard_args
+
+DEFAULT_DIR = "sql/"
+
+parser = ArgumentParser(description=__doc__)
+
+parser.add_argument(
+    "--target", default=DEFAULT_DIR, help="File or directory containing metadata files"
+)
+standard_args.add_log_level(parser)
+
+
+def validate_public_data(metadata, path):
+    """Check if the metadata for public data queries is valid."""
+    is_valid = True
+
+    if metadata.is_public_bigquery() or metadata.is_public_json():
+        if metadata.review_bug() is None:
```

Comment:
```suggestion
        if not metadata.review_bug():
```

With this change, we'll catch empty lists, etc. which should also be flagged as error states.